### PR TITLE
Update jQuery to v3.4.1+

### PR DESCRIPTION
**Pull request by San4es**
Motivation:
https://snyk.io/vuln/SNYK-JS-JQUERY-174006

See also:
https://blog.jquery.com/2019/04/10/jquery-3-4-0-released/
http://blog.jquery.com/2019/05/01/jquery-3-4-1-triggering-focus-events-in-ie-and-finding-root-elements-in-ios-10/

### CODE REVIEW

**Comment by nightskylark** [comment]
@San4es  Did you check this on IE?

**Review by mrjohn0011 — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js [hunk 1]
```diff
@@ -1361,7 +1361,7 @@ QUnit.module("widget sizing render", {}, () => {
         component.repaint();
         $parent.css("transform", "scale(1)");
 
-        assert.equal(component.$element().outerWidth(), initialWidth, "component has correct width");
+        assert.roughEqual(component.$element().outerWidth(), initialWidth, 0.001, "component has correct width");
```
Comment (mrjohn0011) [inline testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js]: Why should we change strict equal to non strict after jquery update?

**Review by mrjohn0011 — DISMISSED** [review]

**Review by alexander-kotov-dx — DISMISSED** [review]

**Comment by BingoRUS** [comment]
We have to update UrlReplacer in hg repo. Please ping Sergey Zvyagin in Teams immediately after merging this PR.

**Comment by San4es** [comment]
>@San4es Did you check this on IE?

@nightskylark thank you for pointing this issue. We have checked and reanimated all failed tests in IE and Edge.

**Review by andrewmakarov — APPROVED** [review]

**Review by nightskylark — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**